### ISSUE TITLE
Lumen 5.2 Issue with Event Service Provider.

### ISSUE THREAD

**Issue by irazasyed** [andersao/l5-repository#117]
There seems to be some issue in Lumen 5.2.

```
PHP Fatal error:  Class 'Illuminate\Foundation\Support\Providers\EventServiceProvider' not found in /Code/lumen/vendor/prettus/l5-repository/src/Prettus/Repository/Providers/EventServiceProvider.php on line 7
```

Any idea?

**Comment by irazasyed** [comment]
So it turns out, Lumen has the event service provider at:
`Laravel\Lumen\Providers\EventServiceProvider`

So if i extend this in Lumen, it works fine. You need to modify to support Lumen and Laravel.

**Comment by irazasyed** [comment]
Sent a PR with a Fix for this - #118
